pynicotine/pynicotine.py

```diff
@@ -992,8 +993,8 @@ def closed_connection(self, conn, addr, error=None):
 
             for i in self.peerconns:
                 if i.conn == conn:
-                    log.add_conn("Connection closed by peer: %(peer)s. Error: %(error)s",
```

Comment:
This reminds me, we should probably remove "by peer". The connection isn't necessary closed by the other user/peer, sometimes we have to do it too.